__tests__/db-tests/get-topics.test.js

```diff
@@ -0,0 +1,40 @@
+const data = require("../../db/data/test-data")
+const app = require("../../app")
+const db = require("../../db/connection")
+const seed = require("../../db/seeds/seed")
+const request = require("supertest")
+
+beforeEach(() => seed(data))
+
+afterAll(() => {
+  db.end()
+})
+
+describe("GET /api/topics", () => {
+  test("Returns a list of all the topics as an array", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toBeInstanceOf(Array)
+      })
+  })
+  test("All returned topic objects have their essential keys", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.topics.forEach((topic) => {
```

Comment:
We need to assert the length of the array, if the array was empty this would pass every time